### PR TITLE
LPS-128510, LPS-128515, LPS-128516, LPS-128517 | Master

### PR DESCRIPTION
**Pull request by Hanlf**
https://issues.liferay.com/browse/LPS-128510
https://issues.liferay.com/browse/LPS-128515
https://issues.liferay.com/browse/LPS-128516
https://issues.liferay.com/browse/LPS-128517

### CODE REVIEW

**Comment by Hanlf** [comment]
ci:test:sf

**Comment by Hanlf** [comment]
ci:test:sf

**Comment by austinchiang** [comment]
**Reviewing...** :hourglass_flowing_sand: :sunglasses: :headphones:

File: modules/apps/depot/depot-test/src/testFunctional/tests/DepotDocuments.testcase [hunk 1]
```diff
@@ -138,6 +138,56 @@ definition {
 		DMDocument.viewDefaultCP();
 	}
 
+	@description = "This test makes sure that the Google Drive Shortcut can be added in a depot."
+	@priority = "4"
+	test EditAndDeleteGoogleDoc {
```
Comment (austinchiang) [inline modules/apps/depot/depot-test/src/testFunctional/tests/DepotDocuments.testcase:143]: @Hanlf I'll push this in for now. Can you move all our Google Docs test cases to a single file in a separate ticket so it's easier to maintain? Release prop should be set to false.

**Comment by austinchiang** [comment]
**Pull request submitted:** https://github.com/liferay-lima/liferay-portal/pull/1575<br></br>Please see changes [here.](https://github.com/austinchiang/liferay-portal/compare/7ae8b536251f~1...7ae8b536251f)

File: modules/apps/depot/depot-test/src/testFunctional/tests/DepotDocuments.testcase (hunk 1 above)
Comment (Hanlf) [reply modules/apps/depot/depot-test/src/testFunctional/tests/DepotDocuments.testcase:143]: Ok, I will do it under [LRQA-65541](https://issues.liferay.com/browse/LRQA-65541).

**Pull request closed without merging**